src/pages/Dashboard/RecentTransactions.tsx

```diff
@@ -27,31 +26,22 @@ type RecentTransactionProps = {
 
 type RecentTransactionsBodyProps = {
   limit: number
-} & (
-  | {
-      txIds: TxId[]
-      filter?: never
-    }
-  | {
-      txIds?: never
-      filter: RecentTransactionFilter
-    }
-)
+} & {
+  filter: RecentTransactionFilter
+}
 
 const defaultFilter = {}
 
 export const RecentTransactionsBody: React.FC<RecentTransactionsBodyProps> = ({
-  txIds: _txIds,
```

Comment:
This was effectively unused - we were selecting txIds in the parent, then passing them down, and re-selecting.
The predicate on why we have this is that some consumers may call `<RecentTransactionsBody />` / `<RecentTransactions />` with `txIds`, in which case we'd skip selection, but none did.